modules/serverless-broker-types.adoc

```diff
@@ -0,0 +1,19 @@
+[id="serverless-broker-types_{context}"]
+= Broker types
+
+There are multiple broker implementations available for use with {ServerlessProductName}, each of which have different event delivery guarantees and use different underlying technologies. You can choose the broker implementation when creating a broker by specifying a broker class, otherwise the default broker class is used. The default broker class can be configured by cluster administrators.
+// TO DO: Need to add docs about setting default broker class.
+
+[id="serverless-using-brokers-channel-based"]
+== Channel-based broker
+
+The channel-based broker implementation internally uses channels for event delivery. Channel-based brokers provide different event delivery guarantees based on the channel implementation a broker instance uses, for example:
+
+* A broker using the `InMemoryChannel` implementation is useful for development and testing purposes, but does not provide adequate event delivery guarantees for production environments.
+
+* A broker using the `KafkaChannel` implementation provides the event delivery guarantees required for a production environment.
+
+[id="serverless-using-brokers-kafka"]
+== Kafka broker
+
+The Kafka broker is a broker implementation that uses Kafka internally to provide at-least once delivery guarantees. It supports any Kafka version, and has a native integration with Kafka for storing and routing events.
```

Comment:
Does it make sense to mention "It supports multiple Kafka versions" instead of "It supports any Kafka version". Typically, we test up to 2 Kafka versions which are supported by the current AMQ Streams that is being used.

Reply:
Updated, thanks!